hack/config.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+usage() {
+    cat <<EOF >&2
+usage:
+
+$0 get-config resourcegroup jsonpath
+
+example:
+    config.sh get-config resourcegroupname .config.HtPasswdPassword
+
+EOF
+    exit 1
+}
+
+if ! rpm -qa | grep -qw jq; then
```

Comment:
remove the jq functionality, output to stdout

Reply:
We have multiple storage accounts, and selecting right one without jq get ambiguous. We can do as you suggest and output to stdout all storage accounts, but we need to chose one name so we can get key and download file. 
Without jq it becomes bash parsing operator to parse json/yaml based on jsonPath.

Alternative?

Reply:
see https://gist.github.com/jim-minter/a108288a581c053cdf2e62f6998b6339

Reply:
we still use jq here, but fine :)